src/api/controllers/adminPermissionsController.js

```diff
@@ -2,7 +2,11 @@ import models from '../models';
 import status from '../../helpers/constants/status.codes';
 import paginateUser from '../../helpers/generate.pagination.details';
 
-const { User } = models;
+const generatePagination = paginateUser;
+
+const {
+  Article, Report, User, Reason
```

Comment:
@Ramadhan0 this change should already be in this file. why is it showing up as part of your changes?